crates/modelardb_server/src/context.rs

```diff
@@ -116,9 +117,7 @@ impl Context {
         let session_runtime = Arc::new(RuntimeEnv::default());
 
         // unwrap() is safe as storage::QUERY_DATA_FOLDER_SCHEME_WITH_HOST is a const containing an URL.
```

Comment:
Maybe remove `storage` from the comment to match the code.

Reply:
Fixed.